patch/dependencies.yaml

```diff
@@ -32,8 +32,10 @@ libcurl_vendor:
   add_run: ["libcurl"]
 intra_process_demo:
   add_host: ["REQUIRE_OPENGL"]
+  # Depends on opencv with gui support, see https://github.com/ros2/demos/blob/8f361a1ece2daae54016959f3ea9ab2e7692ddbb/intra_process_demo/include/image_pipeline/image_view_node.hpp#L58
+  add_run: ["${{ 'libopencv * *qt6*' if not (ppc64le or aarch64 or osx) }}"]
```

Comment:
We have those now on aarch64/osx, and don't care about ppc64, right?

Reply:
Yes, that is a leftover of when it was not there, I will simplify this, thanks!